drivers/drv8825/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "drv8825"
+version = "0.1.0"
```

Comment:
I figured it's best to keep versions in sync between all crates in this repository, for these reasons:
- They all share the same lifecycle. Breaking changes to `step-dir` will require breaking changes to all driver crates, and breaking changes to drivers without breaking changes to `step-dir` are probably rare.
- Syncing versions like that makes it much easier to manage release tags in the repository.

Following this scheme, the first release would be 0.3.0 (unless something else is released in the meantime).

This is just a note to provide some context, no change is required here. Mainly, I'm just writing down my thoughts now, so I'll have an easier time collecting them into proper release procedure documentation later.